docs/modules/maintain-cluster/pages/enterprise-rest-api.adoc

```diff
@@ -0,0 +1,411 @@
+    realms:
+      - name: restRealm
+        authentication:
+          simple:
+            users:
+              - username: 'restuser'
+                password: 'restpassword'
+                roles:
+                  - admin
+        access-control-service:
+          factory-class-name: com.hazelcast.internal.rest.access.DefaultAccessControlServiceFactory
+          properties:
+            decisionFile: '/opt/acl.xml'
+----
+====
+
+== HTTPS Support
+You can configure SSL in the REST API. The configuration options are available to set up SSL as shown in
+the below declarative example:
+
+[tabs]
+====
+XML::
++
+--
+[source,xml]
+----
+<hazelcast>
+    <rest enabled="true">
+        <ssl enabled="true">
+            <client-auth>NEED</client-auth>
+            <ciphers>TLS_RSA_WITH_AES_128_CBC_SHA, TLS_RSA_WITH_AES_128_CBC_SHA256</ciphers>
+            <key-alias>myKeyAlias</key-alias>
+            <key-password>myKeyPassword</key-password>
+            <key-store>/path/to/keystore</key-store>
+            <key-store-password>myKeyStorePassword</key-store-password>
+            <key-store-type>JKS</key-store-type>
+            <key-store-provider>SUN</key-store-provider>
+            <trust-store>/path/to/truststore</trust-store>
+            <trust-store-password>myTrustStorePassword</trust-store-password>
+            <trust-store-type>JKS</trust-store-type>
+            <trust-store-provider>SUN</trust-store-provider>
+            <enabled-protocols>TLSv1.2, TLSv1.3</enabled-protocols>
+            <protocol>TLS</protocol>
+            <certificate>/path/to/certificate</certificate>
+            <certificate-key>/path/to/certificate-key</certificate-key>
+            <trust-certificate>/path/to/trust-certificate</trust-certificate>
+            <trust-certificate-key>/path/to/trust-certificate-key</trust-certificate-key>
+        </ssl>
+    </rest>
+</hazelcast>
+----
+--
+
+YAML::
++
+[source,yaml]
+----
+hazelcast:
+  rest:
+    enabled: true
+    ssl:
+      enabled: true
+      client-auth: NEED
+      ciphers: TLS_RSA_WITH_AES_128_CBC_SHA, TLS_RSA_WITH_AES_128_CBC_SHA256
+      enabled-protocols: TLSv1.2, TLSv1.3
+      key-alias: myKeyAlias
+      key-password: myKeyPassword
+      key-store: /path/to/keystore
+      key-store-password: myKeyStorePassword
+      key-store-type: JKS
+      key-store-provider: SUN
+      trust-store: /path/to/truststore
+      trust-store-password: myTrustStorePassword
+      trust-store-type: JKS
+      trust-store-provider: SUN
+      protocol: TLS
+      certificate: /path/to/certificate
+      certificate-key: /path/to/certificate-key
+      trust-certificate: /path/to/trust-certificate
+      trust-certificate-key: /path/to/trust-certificate-key
+----
+====
+
+
+== JWT Based Authorization
+
+After a successful authentication by making request to token endpoint, you will receive a short-living JWT token with assigned role names as a claim for authorization checks in subsequent REST calls.
```

Comment:
```suggestion
After a successful authentication by making request to token endpoint(`https://localhost:8443/token`), you will receive a short-living JWT token with assigned role names as a claim for authorization checks in subsequent REST calls. 
```

Reply:
I refer the "Obtaining a Token" section here at https://github.com/hazelcast/hz-docs/pull/1097/commits/fc1ae49fe122c52714492f5263a82f3d5d930263